google/cloud/documentai_toolbox/converters/config/converter_helpers.py

```diff
@@ -0,0 +1,485 @@
+# -*- coding: utf-8 -*-
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import re
+import time
+from concurrent import futures
+from typing import List
+
+from google.cloud.documentai_toolbox.converters.config.bbox_conversion import (
+    _convert_bbox_to_docproto_bbox,
+    _get_text_anchor_in_bbox,
+)
+from google.cloud.documentai_toolbox.converters.config.blocks import (
+    load_blocks_from_schema,
+)
+
+from google.cloud.documentai_toolbox import document
+from google.cloud import documentai, storage
+
+
+def get_base_ocr(
+    project_id: str, location: str, processor_id: str, file_bytes, mime_type: str
+) -> documentai.Document:
+    r"""Returns documentai.Document from OCR processor.
+
+    Args:
+        project_id (JSON):
+            Required.
+        location (documentai.Document):
+            Required.
+        processor_id (List[Block]):
+            Required.
+        file_bytes (List[Block]):
+            Required. The bytes of the original pdf.
+        mime_type (List[Block]):
+            Required. usually "application/pdf".
+    Returns:
+        documentai.Document:
+            A documentai.Document from OCR processor.
+
+    """
+    # You must set the api_endpoint if you use a location other than 'us', e.g.:
+
+    client = documentai.DocumentProcessorServiceClient()
+
+    # The full resource name of the processor, e.g.:
+    # projects/project_id/locations/location/processor/processor_id
+    # You must create new processors in the Cloud Console first
+    name = client.processor_path(project_id, location, processor_id)
+
+    # Read the file into memory
+    image_content = file_bytes
+
+    # Load Binary Data into Document AI RawDocument Object
+    raw_document = documentai.RawDocument(content=image_content, mime_type=mime_type)
+
+    # Configure the process request
+    request = documentai.ProcessRequest(name=name, raw_document=raw_document)
```

Comment:
I recommend adding a field mask to this request to specify just the fields you're using. This prevents the large embedded binary content from being downloaded to save on latency and memory.

https://cloud.google.com/document-ai/docs/send-request#online-processor

Reply:
This is just running OCR, and we need everything since we don't know what CDW will use to train the processor from the document.proto

Reply:
Ok, that makes sense.